pkg/pillar/types/zedroutertypes.go

```diff
@@ -320,12 +320,14 @@ type CryptoBlock struct {
 
 // WifiConfig - Wifi structure
 type WifiConfig struct {
-	SSID      string            // wifi SSID
-	KeyScheme WifiKeySchemeType // such as WPA-PSK, WPA-EAP
-	Identity  string            // identity or username for WPA-EAP
-	Password  string            // string of pass phrase or password hash
-	Crypto    CryptoBlock       // encrypted block of items
-	Priority  int32
+	SSID        string            // wifi SSID
+	KeyScheme   WifiKeySchemeType // such as WPA-PSK, WPA-EAP
+	Identity    string            // identity or username for WPA-EAP
+	Password    string            // string of pass phrase or password hash
+	Crypto      CryptoBlock       // encrypted block of items
+	Priority    int32
+	IsCipher    bool
```

Comment:
Please add comment stating that when IsCipher is set, the the CipherBlock contains the Identity and Password and the above Identity and Password strings are empty/ignored.

Reply:
done